### PR TITLE
SLIDESDOC-465 The articles "Supported File Formats" are updated

### PR DESCRIPTION
**Pull request by andreypotapovaspose**
The article "Supported File Formats" (.NET version) is updated: 
- Format descriptions are improved.
- The FODP, MD, and XML formats are added.

### CODE REVIEW

File: net/getting-started/supported-file-formats/_index.md [hunk 1]
```diff
@@ -18,35 +18,33 @@ url: /net/supported-file-formats/
...
+|[PPSM](https://docs.fileformat.com/presentation/ppsm/)|PowerPoint Macro-Enabled Show|{{< emoticons/tick >}}|{{< emoticons/tick >}}| |
+|[POTM](https://docs.fileformat.com/presentation/potm/)|PowerPoint Macro-Enabled Template|{{< emoticons/tick >}}|{{< emoticons/tick >}}| |
+|[ODP/FODP](https://docs.fileformat.com/presentation/odp/)|OpenDocument Presentation|{{< emoticons/tick >}}|{{< emoticons/tick >}}| |
 |[OTP](https://docs.fileformat.com/presentation/otp/)|OpenDocument Presentation Template|{{< emoticons/tick >}}|{{< emoticons/tick >}}| |
-|[TIFF](https://docs.fileformat.com/image/tiff/)|Saves the document as a Single or Multi-Page TIFF Image| |{{< emoticons/tick >}}| |
-|[EMF](https://docs.fileformat.com/image/emf/)|Enhanced Metafile Format Plus Extensions| |{{< emoticons/tick >}}| |
+|[TIFF](https://docs.fileformat.com/image/tiff/)|Tag Image File Format| |{{< emoticons/tick >}}| |
+|[EMF/EMF+](https://docs.fileformat.com/image/emf/)|Enhanced Metafile Format (Plus Extensions)| |{{< emoticons/tick >}}| |
 |[PDF](https://docs.fileformat.com/pdf/)|Portable Document Format| |{{< emoticons/tick >}}| |
```
Comment (yury-knml) [inline net/getting-started/supported-file-formats/_index.md:39]: PDF import = load?

File: net/getting-started/supported-file-formats/_index.md [hunk 2]
```diff
@@ -18,35 +18,33 @@ url: /net/supported-file-formats/
...
+|[GIF](https://docs.fileformat.com/image/gif/)|Graphics Interchange Format| |{{< emoticons/tick >}}| |
+|[BMP](https://docs.fileformat.com/image/bmp/)|Device Independent Bitmap| |{{< emoticons/tick >}}| |
+|[SVG](https://docs.fileformat.com/page-description-language/svg/)|Scalable Vector Graphics| |{{< emoticons/tick >}}| |
 |[SWF](https://docs.fileformat.com/page-description-language/swf/)|Small Web Format| |{{< emoticons/tick >}}| |
-|[HTML](https://docs.fileformat.com/web/html/)|HTML Format| |{{< emoticons/tick >}}| |
-|[XAML](https://docs.fileformat.com/web/xaml/)|XAML Format| |{{< emoticons/tick >}}| |
-
-
-
+|[HTML](https://docs.fileformat.com/web/html/)|Hypertext Markup Language| |{{< emoticons/tick >}}| |
```
Comment (yury-knml) [inline net/getting-started/supported-file-formats/_index.md:47]: HTML import = load?

File: net/getting-started/supported-file-formats/_index.md (hunk 1 above)
Comment (andreypotapovaspose) [reply net/getting-started/supported-file-formats/_index.md]: Corrected.

File: net/getting-started/supported-file-formats/_index.md (hunk 2 above)
Comment (andreypotapovaspose) [reply net/getting-started/supported-file-formats/_index.md]: Corrected.

**Review by yury-knml — APPROVED** [review]

**Pull request merged**